sandbox/libs/authn/src/main/java/org/opensearch/authn/Subject.java

```diff
@@ -28,5 +29,5 @@ public interface Subject {
      * throws SubjectNotFound
      * throws SubjectDisabled
      */
-    void login(final AuthenticationToken token);
+    void login(final AuthenticationToken token) throws BadCredentialsException;
```

Comment:
Lets update `BadCredentialsException` to be off of RuntimeException to avoid all these method annotations at least for the moment.  Even if we do keep it as a checked exception, I think we should have our own base type, 'AuthenticationException' or something along those lines that all exceptions are based from

Reply:
I swapped them all over to just runtimeexceptions if this is not what you meant let me know.